### ISSUE TITLE
Automatic tagging does not work on `external-bridged-provider` template

### ISSUE THREAD

**Issue by kangasta** [pulumi/ci-mgmt#1437]
The `tag_release_if_labeled_needs_release` job on the `main.yml` workflow uses `pulumi/action-release-by-pr-label`, which assumes that the repository has access to [pulumi/release-bot](https://github.com/pulumi/release-bot). Third party plugins likely do not have access to the release-bot and thus automatic tagging will not work.

Could this part of the workflow be updated so that the required commands (`git tag`, `git push`, ...) would be executed in GitHub Actions (instead of making HTTP call to release-bot) when using `external-bridged-provider` template?
